### ISSUE TITLE
Make '--verbose' parameter more verbose

### ISSUE THREAD

**Issue by satano** [leonardochaia/dotnet-affected#25]
When I use `--verbose` parameter, it show some more info. but this is just some _statistical_ info: count of changed files and projects. This itself is fine.

`describe` command is interesting, because it writes which projects are changed and which are affected by changes. But this command just prints this information, it does not saves `.proj` file.

It would be nice to have `describe`'s command information in standard output when using `--verbose`. This is good for logs, where we can immediately see which projects have changed. In CI builds, there is hard (or no at all) possibility to inspect output `.proj` file, but logs are available for a long time.

**Comment by leonardochaia** [comment]
Hi @satano ! good to know somebody else is trying this out. I would love to hear any feedback whatsover.

I was looking at our CI pipeline and we are indeed printing the changed projects, using `cat`, so yeah I think this would be a nice addition. We used to have it on v1.0 and I think we've lost it when we refactored it. We may need to introduce a verbosity level tho.

That being said, with the current `v2.0.0-preview-1` there are two workarounds:
1. `cat` the output file:
```bash
dotnet affected --verbose
cat affected.proj
```
2. Run `dotnet affected` twice (has performance cost)
```bash
dotnet affected -f text --dry-run --verbose
```

I do agree that this would be a nice addition, so I'll convert this to a feature request.
I think we should be able to log Project Names / Paths depending on verbosity level.

**Comment by leonardochaia** [comment]
The reason why they were split is because we didn't wanna output a bunch of information by default.

Originally it used to print everything together with the generation of the traversal file, [source](https://github.com/leonardochaia/dotnet-affected/tree/v1.0.0-preview-2#generate-a-project-to-build-all-affected-projects)

When 2.0 was released, I decided to provide a `describe` command and do the generation by default.

My vote is to print the changed/affected projects, the same way `describe` does it when `--verbose` is present, but I am open to suggestion.
